docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md

```diff
@@ -0,0 +1,82 @@
+---
+id: using-existing-elasticsearch
+title: "Using existing Elasticsearch"
+description: "Learn how to use an existing elasticsearch instance in Camunda 8 Self-Managed deployment."
+---
+
+Camunda 8 Self-Managed has two different types of applications: Camunda applications (Operate, Optimize, Tasklist, etc.) and non-Camunda applications (such as Keycloak and Elasticsearch). For more details, review the [architecture](../../../platform-architecture/overview.md) documentation for more information on the different types of applications.
```

Comment:
> Camunda 8 Self-Managed has two different types of applications: Camunda applications (Operate, Optimize, Tasklist, etc.) and non-Camunda applications (such as Keycloak and Elasticsearch)

i would use the terms used in the architecture slide (Camunda Component and Dependency Components).

Reply:
done